frontend/src/TestUtils.ts

```diff
@@ -10,7 +10,7 @@ export function createMessageForTesting(
   let directMessageID = null;
   let toUserName = null;
   if (player2Id) {
-    directMessageID = `${player1Id}:${player2Id}`;
+    directMessageID = [player1Id, player2Id].sort().join(':');
```

Comment:
the order of the ids should be an invariant—and the lack of enforcement here was breaking some tests

Reply:
oo good catch